app/assets/stylesheets/govuk_publishing_components/components/_previous-and-next-navigation.scss

```diff
@@ -54,6 +69,7 @@
   margin-bottom: 1px;
   height: .482em;
   width: .63em;
+  fill: $govuk-secondary-text-colour;
```

Comment:
Why a text colour rather than the standard border colour? $govuk-border-colour

Reply:
The border colour is a slightly paler grey, which means the arrow can't be seen very easily:

<img width="196" alt="Screenshot 2020-03-03 at 12 04 13" src="https://user-images.githubusercontent.com/29889908/75774131-2a2ee900-5d47-11ea-94c9-e2aae3eb4691.png">

However, the govuk-colour("mid-grey") is the same colour, so I can use the mixin instead as it's not text content.